doc/content/source/meshgenerators/ParsedElementIDMeshGenerator.md

```diff
@@ -0,0 +1,27 @@
+# ParsedElementIDMeshGenerator
+
+
+## Description
+
+`ParsedElementIDMeshGenerator` is a mesh generator that wraps an existing input mesh and adds one or
+more extra element integer id (eeid) to the mesh, where each eeids corresponds to user-defined values.
```

Comment:
Is it really a user-defined value? It's more like the user defines the amalgamation scheme, and then the IDs flow naturally from that. I'd suggest to remove "where each eeids corresponds to user-defined values" for clarity if that is the case because it sort of implies that the user is setting the numbers themeslves.